test/sequential/test-repl-persistent-history.js

```diff
@@ -147,6 +150,20 @@ const tests = [{
   test: [UP, UP, UP, CLEAR],
   expected: [prompt, convertMsg, prompt, prompt + '\'=^.^=\'', prompt]
 },
+{
+  before: function() {
+    try {
+      fs.mkdirSync(historyPathFail);
+    } catch (err) {}
```

Comment:
I would also not catch this one, but it is less important, what were you thinking of trying to avoid?

Actually since you're using a different file, catching both will technically be fine.